src/fare-contracts/__tests__/sort-fc-reservation-by-validity-creation.test.ts

```diff
@@ -112,7 +112,7 @@ describe('Sort by Validity', () => {
   it('Reservation should be first if reservation is being processing', async () => {
     const fcOrReservations: (FareContract | Reservation)[] = [
       mockupFareContract('1', 'valid', -1),
-      mockupReservation('3', 'reserving', 0),
+      mockupReservation('3', 'reserving', 1),
```

Comment:
It seems to me that it was kinda the point that the reservation and farecontract had the same timestamp, to test that the reserving reservation should be sorted first.